x-pack/agent/pkg/agent/application/fleet_gateway_test.go

```diff
@@ -242,13 +251,72 @@ func TestFleetGateway(t *testing.T) {
 		}
 	})
 
+	t.Run("send event and receive no action", withGateway(agentInfo, func(
+		t *testing.T,
+		gateway *fleetGateway,
+		client *testingClient,
+		dispatcher *testingDispatcher,
+		scheduler *scheduler.Stepper,
+		rep repo.Backend,
+	) {
+		rep.Report(&testStateEvent{})
+		received := ackSeq(
+			client.Answer(func(headers http.Header, body io.Reader) (*http.Response, error) {
+				cr := &request{}
+				content, err := ioutil.ReadAll(body)
+				if err != nil {
+					t.Fatal(err)
+				}
+				err = json.Unmarshal(content, &cr)
+				if err != nil {
+					t.Fatal(err)
+				}
+
+				require.Equal(t, 1, len(cr.Events))
+
+				// TODO: assert no events
```

Comment:
remove `TODO`

Reply:
this diff is weird. todo is removed and json indentation fixed but diff still shows something stale. Oh my github